src/main/java/com/nutomic/syncthingandroid/syncthing/SyncthingRunnable.java

```diff
@@ -88,8 +91,7 @@ public void run() {
         try {
             ProcessBuilder pb = new ProcessBuilder("chmod", "+x", mSyncthingBinary);
             Process p = pb.start();
-            if (p != null)
-                p.waitFor();
+            p.waitFor();
```

Comment:
Are you sure this isn't possible? I think I added it because of a crash submitted to the play store, but I'm not sure.